src/js/events.js

```diff
@@ -208,15 +211,17 @@ export class RegularViewEventModel extends RegularVirtualTableViewModel {
      * @param {*} metadata
      */
     _on_resize_column(event, element, metadata) {
+        const {_virtual_x, size_key} = metadata;
         const start = event.pageX;
-        element = this.table_model.header.get_column_header(metadata._virtual_x);
-        const width = this._column_sizes.indices[metadata.size_key];
-        const move = (event) => this._on_resize_column_move(event, element, start, width, metadata);
+        const header_x = _virtual_x + element.colSpan - 1;
```

Comment:
This is the source of the bug - where previously we used `_virtual_x`, he now use `header_x` which takes into account a label's `colSpan` when determining which data column they correspond to.